code/__DEFINES/is_helpers.dm

```diff
@@ -293,15 +293,22 @@ GLOBAL_LIST_INIT(glass_sheet_types, typecacheof(list(
 
 #define isshuttleturf(T) (length(T.baseturfs) && (/turf/baseturf_skipover/shuttle in T.baseturfs))
 
+#define isbook(O) (is_type_in_typecache(O, GLOB.book_types))
+
+GLOBAL_LIST_INIT(book_types, typecacheof(list(
+	/obj/item/book,
+	/obj/item/spellbook,
+	/obj/item/storage/book)))
+
+#define IS_EDIBLE(O) (O.GetComponent(/datum/component/edible))
```

Comment:
duplicate def

Reply:
Im the one who moved it originally too, man thats ironic.

removed in 3814797